src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
 
```

Comment:
3. Overall:
   - The code structure and functionality seem fine, but it's always recommended to provide proper error handling for exceptions that can occur. Ensure that all methods are properly tested to handle different scenarios.